Formula/s/simh.rb

```diff
@@ -27,16 +26,22 @@ class Simh < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "9ff6b20047316c5efd516276d7029475379b026f43b446e1707e488cff52be10"
   end
 
-  depends_on "libpng"
-  depends_on "sdl2"
+  # https://github.com/simh/simh/issues/1163
+  deprecate! date: "2022-05-12", because: "changed to a non-free license in master branch"
```

Comment:
We use these dates to get an idea when something should be disabled so I'd just set it to today. The original license change date isn't particularly meaningful to us given the current version is still unaffected.